src/sagemaker/fw_utils.py

```diff
@@ -74,6 +74,9 @@
         "2.6",
         "2.6.0",
         "2.6.2",
+        "2.7",
+        "2.7.0",
```

Comment:
There are no TF 2.7.0 SM Training DLCs, so this can be safely removed from this list.